server/src/main/java/org/opensearch/index/search/stats/SearchStats.java

```diff
@@ -128,6 +140,12 @@ private Stats(StreamInput in) throws IOException {
             suggestCount = in.readVLong();
             suggestTimeInMillis = in.readVLong();
             suggestCurrent = in.readVLong();
+
+            if (in.getVersion().onOrAfter(Version.V_3_0_0)) {
+                pitCount = in.readVLong();
+                pitTimeInMillis = in.readVLong();
+                pitCurrent = in.readVLong();
+            }
```

Comment:
Aren't we targeting 2.3?

Reply:
yes we are targeting for 2.3.
Complete details in this thread: https://github.com/opensearch-project/OpenSearch/pull/4030#discussion_r935815185